kube/resources/pod-sweeper.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airbyte-pod-sweeper
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      airbyte: pod-sweeper
+  template:
+    metadata:
+      labels:
+        airbyte: pod-sweeper
+    spec:
+      containers:
+        - name: airbyte-pod-sweeper
+          image: bitnami/kubectl
+          imagePullPolicy: IfNotPresent
+          command:
+            - /bin/sh
+            - -c
+            - while :; do kubectl get pod | grep Completed | awk '{print $1}' | xargs --no-run-if-empty kubectl delete pod | xargs --no-run-if-empty echo `date "+%Y-%m-%d %H:%M:%S"`; sleep 300; done
```

Comment:
This doesn't handle namespaced airbyte deployments. It also doesn't limit itself to just pods that were created as part of Airbyte.

Probably the easiest way to do this is to add a label to all connector pods / command fetcher pods that makes it easy to `kubectl get` them by this label. Then you can also insert the namespace used in a similar way that we pulled it in as an env variable before, and use that to scope the search for pods.